### ISSUE TITLE
Create some graphics for things

### ISSUE THREAD

**Issue by lpmi-13** [school-as-code-testing/school-as-code#4]
some of the things we have in DESIGN_DOCS would benefit from a bit more context. It would be nice to have a graphic that shows the interaction of external factors during even just basic "work as a programmer". Particularly in reference to the line

> rhetorical situation: there is no escape! every act is born in a context that it in turn helps to define.

in the first principles. A visual with all the other stuff like "access to a network connection, needs from stakeholders, work/life balance, etc" would help to give some concrete examples of the practical manifestation of phrases like the above.

**Comment by colevandersWands** [comment]
> access to a network connection, needs from stakeholders, work/life balance, etc

These are good and missing in the doc.  did you want to add them or integrate them in existing principles?

**Comment by lpmi-13** [comment]
I think more examples in general in that part of the doc would be good. Then again, if we had a number of examples for each of those, it might get a bit unwieldy very quickly...I might be leaning more towards leaving it as is. I'm just conscious of having those principles guide actions, and if it's not immediately clear what types of actions would be manifestations of those principles, then there's a danger that they could be perceived as very buzzword-y.

**Comment by colevandersWands** [comment]
it could include examples and stay tidy-looking if the examples are links or collapsed in `<details>` tags?  the buzzword-y risk is more important than unwieldiness imo.

**Comment by lpmi-13** [comment]
yeah, that could work. It's in its own sub-doc anyway, so making it get slightly bigger (and collapsable in the UI) is probably not a big deal.
